include/boost/dynamic_bitset/dynamic_bitset.hpp

```diff
@@ -1622,6 +1624,17 @@ inline bool operator>=(const dynamic_bitset<Block, Allocator>& a,
     return !(a < b);
 }
 
+//-----------------------------------------------------------------------------
+// hash operations
+
+template <typename Block, typename Allocator>
+inline std::size_t hash_value(const dynamic_bitset<Block, Allocator>& a)
+{
+    std::size_t res = hash_value(a.m_num_bits);
+    hash_combine(res, a.m_bits);
```

Comment:
`hash_combine` is not an ADL customization point like `hash_value` is. Qualify as `boost::hash_combine` here.

Reply:
Thanks - pushed a fix.